test/e2e-go/cli/goal/expect/tealConsensusTest.exp

```diff
@@ -84,16 +88,18 @@ if { [catch {
         " - pass -" { puts "small-sig dryrun pass" }
         "REJECT" { ::AlgorandGoal::Abort $expect_out(buffer) }
         "static cost budget" { ::AlgorandGoal::Abort $expect_out(buffer) }
+        eof { ::AlgorandGoal::Abort $expect_out(buffer) }
     }
 
-    teal "$TEST_ROOT_DIR/slow-sig.teal" 2 1 20001
+    teal "$TEST_ROOT_DIR/slow-sig.teal" 4 1 20001
```

Comment:
see the original [PR](https://github.com/algorand/go-algorand/pull/2126/files#diff-9d9a133c6ac373272816e6541cd66e3238317f403f743da0215d489f25341025) that added teal v4 with back jumps and subroutines so check does not enforce cost budget